src/pages/home/report/ReportActionsView.js

```diff
@@ -298,6 +298,9 @@ class ReportActionsView extends React.Component {
         Report.readOldestAction(this.props.report.reportID, oldestActionSequenceNumber);
     }
 
+    /**
+     * Scrolls down to the bottom of the chat when the new messages dialog is clicked.
```

Comment:
Just to clarify, are we planning to alter this behavior in the future?

Reply:
I don't really think this change is needed. It won't really prevent anyone from changing anything if they are determined to do so.

Reply:
Also we should not say where a function will be used in a comment. It could be used in multiple places and you can search to see where it is actually used.

Reply:
Yeah that makes sense - I just added a use case since I saw it done in the [comment below it ](https://github.com/Expensify/App/blob/srikar-jumpToUnreadComment/src/pages/home/report/ReportActionsView.js#L310) but yeah new use cases could be made for that method. I think that it just adds another layer that someone would have to make to change this behavior and would be more easily caught since it's a comment which is clearer than a method name. I also don't think the change has any drawbacks but if you don't think it's necessary at all, I can close out this PR.

Reply:
And I'm not sure @MariaHCD, I was talking to Marc about this last week and understood that we're trying to implement bidirectional scrolling which would help in situations like this but I'm not sure if we have plans to change the functionality of when someone clicks "New Messages"

Reply:
Personally, I think we should close this. We have no way of knowing whether this comment will help avoid any future or past situation. The content can also be extracted by looking at the method name and then looking at where it's used.